gradle/libs.versions.toml

```diff
@@ -0,0 +1,35 @@
+[versions]
+aspectj-version = "1.9.9.1"
+byteBuddy-version = "1.12.8"
+junit4-version = "4.13.2"
+slf4j-version = "1.7.35"
+spring-version = "5.3.19"
+
+[libraries]
+assertj = { module = "org.assertj:assertj-core", version = "3.22.0" }
```

Comment:
Could it be of advantage to move all version numbers to the [versions]-part?

Reply:
I would only move them there if we intend to reference or reuse them. Otherwise, you'll get a ton of versions that are only used once and I feel that that just clutters files